be/src/common/config.h

```diff
@@ -823,6 +823,10 @@ CONF_Int32(starlet_cache_dir_allocate_policy, "0");
 // Buffer size in starlet fs buffer stream, size <= 0 means not use buffer stream.
 // Only support in S3/HDFS currently.
 CONF_Int32(starlet_fs_stream_buffer_size_bytes, "131072");
+CONF_mBool(starlet_use_star_cache, "false");
+CONF_mString(starlet_star_cache_mem_size_limit, "4%");
```

Comment:
completely disable mem usage?

Reply:
no, starcache's meta needs this, just like tablet meta cache

Reply:
this is strange. Assume the meta may require 100MB mem, when I set to less than 100MB, the starcache can't even work? or will just use about let's say 70MB, and then out of space. It leaves the question to the user, how much mem I have to set if I want to give about X MB to cache, do  I need to provide X / (1 - y%) overhead or not? what's the overhead of y%?

Reply:
will default to 0 when next starlet version is ready